cobra/io/sbml.py

```diff
@@ -312,12 +312,15 @@ def _sbml_to_model(doc, number=float, f_replace=F_REPLACE,
                 raise Exception("Conversion of SBML fbc v1 to fbc v2 failed")
 
     # Model
-    cobra_model = Model(model.getId())
+    model_id = model.getIdAttribute()
+    if not libsbml.SyntaxChecker.isValidSBMLSId(model_id):
+        logging.error("Model id '%s' is no valid SId." % model_id)
```

Comment:
Change to `logging.error("'%s' is not a valid SBML 'SId'." % value)` as used below.